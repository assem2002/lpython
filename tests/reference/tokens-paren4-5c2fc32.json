{
    "basename": "tokens-paren4-5c2fc32",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/errors/paren4.py",
    "infile_hash": "10d0c8281aba40d1910e24ea824a43117ba9e95d518053323278284d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "tokens-paren4-5c2fc32.stderr",
    "stderr_hash": "d1b1369b365f63a4a700a34fce46c2785ff5b52b99f9ab87250e1aef",
    "returncode": 1
}