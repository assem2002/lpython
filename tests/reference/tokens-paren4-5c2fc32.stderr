tokenizer error: Parentheses was never closed
 --> tests/tokens/errors/paren4.py:6:1
  |
6 | 
  | ^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
